src/dsl/lower.rs

```diff
@@ -82,25 +95,15 @@ impl Lowerer {
                         abstract_: false,
                         name,
                         generics: vec![],
-                        superclass,
+                        superclass: Some(ast::Type::simple_path("StatefulWidget")),
                         mixins: vec![],
                         interfaces: vec![],
                         members: class_members,
                     }));
 
-                    superclass = Some(Node::new(ast::Type::Path(
-                        ast::Qualified::simple("State"),
-                        vec![
-                            Node::new(ast::Type::Path(
-                                ast::Qualified { prefix: None, name },
-                                vec![],
-                            )),
-                        ],
-                    )));
                     name = Symbol::intern(&state_name);
```

Comment:
Doesn't this make stateful widgets have the wrong supertrait? I would not use mutation here but add a match later.